storm-core/src/ui/public/templates/topology-page-template.html

```diff
@@ -313,6 +320,10 @@
         </th>
         <th class="header">Last error
         </th>
+		<th class="header">
```

Comment:
@HeartSaVioR 
Code style shoud be checked before commit , shall we provide ```.editorconfig ``` with ````git pre-commit hooks```  ?